examples/frameworkless-on-kubernetes-example/readme.md

```diff
@@ -0,0 +1,69 @@
+# Frameworkless Rest API on Kubernetes 
+
+
+The purpose of this example is to demonstrate the following:
+
+- How you can use the kubernetes-annotations to configure a web app and the services.
+- How you can end-to-end test the application.
+- How you can trigger an image build after the compilation.
+
+
+The application is using:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>kubernetes-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+Which contains the annotation processors that detect web applications.
+
+The [App.java](src/main/java/io/dekorate/example/fwlonk8s/App.java) is annotated with `@KubernetesApplication` which triggers the resource generation.
+
+```
+@KubernetesApplication(
```

Comment:
We need to align this with the current content of the class  (e.g. remove name, imagePullPolicy)